Ja postroute addstock [JA-1]

arsoljon opened the pull request:
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]


* **What kind of change does this PR introduce?** (Bug fix, new feature...)

Added the post route, '/stocks/addStock'. 

* **What is the current behavior? What does the code do before you made the new changes?** 

The current behavior does not allow users to access any routes. 

* **What is the new behavior?**

The new behavior allows posting a new stock to a particular user.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

The only breaking change, is to make note of this post route. There was nothing new installed. 

* **Other information**:

arsoljon committed:
created the post route for adding a stock. Has route '/stocks/addStock'

arsoljon committed:
cleaned the code

Codubee commented on index.js:
```diff
@@ -0,0 +1,24 @@
+
+
+
+app.post('/stocks/addStock', function(req, res){
```
Consider making the path name shorter here. I feel like the path '/stocks/addStock' is a bit long.

arsoljon committed:
Update index.js

Commented out the listen method at the end of file because I am not sure if there will be conflicts with different pull request that are working on the same file.

Codubee reviewed (dismissed):
Everything else looks good. Just gotta do that one thing.

arsoljon committed:
Update index.js

Shortened the length for the path name on line 8 .
